components/OnboardingScreens/OnboardingScreens.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import OnboardingScreen from './OnboardingScreen';
+import FinalOnboardingScreen from './FinalOnboardingScreen';
+import { TURQUOISE, BLUE, PURPLE } from '../../common/colors';
+import Swiper from 'react-native-swiper';
+
+interface OnboardingScreenProps {
+    navigation: any;
```

Comment:
I think one of you asked about types for react navigation, this page of the docs looks helpful for implementing them:

https://reactnavigation.org/docs/typescript/

Reply:
Additionally, I think it would be best if the onboarding pages led to the register screen, rather than the log in screen.